healpix_alchemy/types.py

```diff
@@ -28,6 +28,15 @@ def process_bind_param(self, value, dialect):
             value = int(value)
         return value
 
+    def to_moc(self, rangeSet, nside, index):
+        healpixSet = np.unique(np.concatenate([np.arange(start, stop, 1) for
+                                               (start, stop) in rangeSet]))
+        if index.lower() == "ring":
```

Comment:
Why do we need to support ring indexing at all? If we are trying to initialize a MOC object from an array of Tile instances, then the ranges represented by the Tiles will always use nested ordering.

Reply:
I added it just for completeness, but have no issue with just using nested